main.go

```diff
@@ -51,10 +51,22 @@ func run(cmd *cobra.Command, args []string) {
 		slog.Error("Espresso Reader couldn't connect to the database", "error", err)
 		os.Exit(1)
 	}
+	max_attempts := 10
+	var config *model.NodeConfig[model.NodeConfigValue]
+	for i := 1; i <= max_attempts; i++ {
+		config, err = repository.LoadNodeConfig[model.NodeConfigValue](ctx, database, model.BaseConfigKey)
+		if err == nil {
+			break
+		}
+		slog.Warn("Failed to load configuration, retrying...", "attempt", i, "error", err)
+		if i < max_attempts {
+			time.Sleep(3 * time.Second)
+		}
+	}
```

Comment:
Why is there a max_attempts?

Reply:
I mean, why not just leave it there forever retrying to load the config?

Reply:
I don’t like the idea of hanging or waiting forever... it’s more about failing fast.
In a scenario where we wait forever, we end up spending more time detecting the problem...
However, in the case of in the middle of running an input reader, it’s better to wait indefinitely because the application might be experiencing an internet connection issue.
The difference here is that we are starting the node.

Reply:
By the way, on the second attempt, the config was successfully loaded.

Reply:
Ok, I see your point, especially for the CI case. I think this should ideally be configurable, because for me it makes sense in certain scenarios to keep trying until it's there. But it's definitely something we can leave to another time!